README.md

```diff
@@ -1,284 +1,305 @@
-    The checks are included in gradle's *check* task (you can run them by executing `./gradlew check` command).
+PI_AAD users, excluding third parties, will be stored in Azure B2C. Azure B2C provides OIDC which is used in our frontend to authenticate the users.
 
-  * pmd
+Account management interacts with Azure B2C - storing the user when they are created, updating them when their role changes, or deleting them if they have expired.
 
-    https://docs.gradle.org/current/userguide/pmd_plugin.html
+## System Admin Accounts
 
-    Performs static code analysis to finds common programming flaws. Included in gradle `check` task.
+There is a limit to the number of system admin accounts that can be created. This is set using an environment variable: MAX_SYSTEM_ADMIN_ACCOUNTS.
 
+If this is not set, the default maximum number of accounts is 4. The request will be rejected if an attempt is made that causes it to go over this limit.
 
-  * jacoco
+## List Sensitivity checking
 
-    https://docs.gradle.org/current/userguide/jacoco_plugin.html
+### Verified Users
 
-    Provides code coverage metrics for Java code via integration with JaCoCo.
-    You can create the report by running the following command:
+If a publication is PRIVATE or CLASSIFIED, account management is used to validate whether the user has permission to see the publication.
 
-    ```bash
-      ./gradlew jacocoTestReport
-    ```
+For PRIVATE, as long as the user exists in our database, then they are able to see the publication.
 
-    The report will be created in build/reports subdirectory in your project directory.
+For CLASSIFIED, rules are applied based on the List Type and Provenance of the user. For reference, the provenance is which IDAM the user came from (PI_AAD, CFT_IDAM or CRIME_IDAM). The mappings of which provenance can see which list type is defined in the [enum file](./src/main/java/uk/gov/hmcts/reform/pip/account/management/model/ListType.java)
 
-  * io.spring.dependency-management.
+### Third Party Users
 
-    https://github.com/spring-gradle-plugins/dependency-management-plugin
+For third party users, a similar process is applied however is based on List Type + Third Party role. This is also defined in: [enum file](./src/main/java/uk/gov/hmcts/reform/pip/account/management/model/ListType.java)
 
-    Provides Maven-like dependency management. Allows you to declare dependency management
-    using `dependency 'groupId:artifactId:version'`
-    or `dependency group:'group', name:'name', version:version'`.
+## Account Application Process
 
-  * org.springframework.boot
+When a user applies for a new PI_AAD verified account they go through an approval process. Account management deals with all CRUD operations associated with this process.
 
-    http://projects.spring.io/spring-boot/
+On Approval / Rejection, the uploaded ID file will be deleted.
 
-    Reduces the amount of work needed to create a Spring application
+## Expired / Inactive accounts
```

Comment:
"Accounts" rather than "accounts" in heading?

Reply:
Updated